client/web/src/components/RadioButtons.tsx

```diff
@@ -50,19 +50,25 @@ interface Props {
      * id of the currently selected RadioButtonNode.
      */
     selected?: string
+
+    /**
+     * name of the field that these radio buttons select. Use a unique name for
+     * each group of radio buttons in a form.
+     */
+    name?: string
```

Comment:
Seems to me that this should be a required prop.
```suggestion
    name: string
```

Reply:
Yeah, I was unsure whether to do that in this PR or not, since I'll have to update the other places as well, but that should be quick. I'll do that.

Reply:
Looks like only two places, surprisingly 🤔

Reply:
Fixed in 728bacb